.github/workflows/ci-arm-build.yml

```diff
@@ -0,0 +1,67 @@
+name: CI ARM64 Build and Push
+
+on:
+  # push:
```

Comment:
rm commetns?

Reply:
no, reason: this should become automatic in one of the next PRs, I need this in master first of all.